lib/rebel/sql.rb

```diff
@@ -304,10 +308,26 @@ def equal(l, r)
       "#{name_or_value(l)} = #{name_or_value(r)}"
     end
 
+    def order(l, r)
+      "#{name(l)} #{value(raw(r.to_s))}"
```

Comment:
Code smell: this doesn't match the method name and is too generic. Plus `value(raw())` doesn't make sense (see `value`). The argument names also aren't explicit, copy-pasted from `equal` above. When `r` is `nil` this generates an extra space.

This method should generate `something [ASC|DESC]` and nothing else, with no extra space. Its arguments are thus `column` and `direction`, maybe shortened to `c` and `d`. It should also mimic the SQL signature, defaulting `direction` to `nil`. The name should be `order_term`, as `order` suggests something else.

There is also the case of passing a Raw directly. Such a Raw could be created by calling a `asc` or `desc` method on a Raw, like `name(:foo).asc` or `by(:bar, :baz).desc`. In the future, `Raw` will be increasingly replaced by stronger types that will allow for additional checks and limit available methods.